projects/addon-doc/components/aside/aside.component.less

```diff
@@ -0,0 +1,16 @@
+:host._empty {
+    display: none;
+}
+
+.t-link {
+    text-decoration: none;
+    color: var(--tui-text-01);
```

Comment:
These styles are unnecessary after you get rid of `button`.

Reply:
fixed